app/views/partials/client/item.blade.php

```diff
@@ -1,7 +1,7 @@
 <td>{{ $client->authority['name'] }}</td>
-<td>{{ chunk_split($client->api['basic_key'], 20, '</br>') }}</td>
-<td>{{ chunk_split($client->api['basic_secret'], 20, '</br>') }}</td>
-<td>
+<td>{{ chunk_split($client->api['basic_key'], 40, '') }}</td>
+<td>{{ chunk_split($client->api['basic_secret'], 40, '') }}</td>
+<td><center>
```

Comment:
I think the `center` tag is frowned upon because it's more of a styling thing so it belongs in the CSS.